lib/binding_web/exports.json

```diff
@@ -24,6 +24,7 @@
   "_memcmp",
   "_memcpy",
   "_memmove",
+  "_memset",
```

Comment:
This fixes the issue from running `tree-sitter-python`, see [my comment](https://github.com/tree-sitter/tree-sitter/pull/1913#issuecomment-1281130984)